src/ui/components/area-list.component.ts

```
@@ -0,0 +1,185 @@
+        if (!this.listHostDiv || !this.listHostDiv.isConnected) {
+            this.element.innerHTML = ''; // Clear "no areas" message or old list
+            this.listHostDiv = document.createElement('div');
+            this.listHostDiv.id = 'area-list'; // Matching ID from original HTML structure
+            this.appendChild(this.listHostDiv);
+        }
+    }
+
+    private renderFullList(): void {
+        if (this.areas.length === 0) {
+            this.element.innerHTML = '<p>Noch keine Bereiche für diesen Tresen erfasst.</p>';
+            if (this.listHostDiv && this.listHostDiv.isConnected) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)